src/main/java/racingcar/InputHandler.java

```diff
@@ -0,0 +1,61 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputHandler {
+    public static List<String> getCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String input = Console.readLine();
+        validateCarNamesInput(input);
+
+        List<String> carNames = new ArrayList<>();
+        for (String carName : input.split(",")) {
+            carName = carName.strip();
+            validateCarName(carName);
+            carNames.add(carName);
+        }
+
+        return carNames;
+    }
+
+    public static int getTotalRaceCount() {
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        String input = Console.readLine();
+        return parseAndValidatePositiveInteger(input);
+    }
+
+    private static void validateCarNamesInput(final String input) {
+        if (input == null || input.isBlank()) {
+            throw new IllegalArgumentException("경주할 자동차 이름을 입력하세요.");
+        }
+        if (input.split(",").length == 0) {
+            throw new IllegalArgumentException("쉼표 사이에 이름이 작성되어야 합니다.");
+        }
+    }
+
+    private static void validateCarName(final String carName) {
+        if (carName.isBlank()) {
+            throw new IllegalArgumentException("쉼표 사이에 이름이 작성되어야 합니다.");
+        }
+        if (carName.contains(" ")) {
+            throw new IllegalArgumentException("이름에 공백이 포함될 수 없습니다.");
+        }
+        if (carName.length() > 5) {
+            throw new IllegalArgumentException("이름은 5글자 이하로 작성 가능합니다.");
+        }
+    }
+
+    private static int parseAndValidatePositiveInteger(String input) {
```

Comment:
```suggestion

    private static int parseAndValidatePositiveInteger(final String input) {
```

파라미터 final 사용에 대한 일관성이 있으면 좋을 것 같습니다!

Reply:
저도 리팩터링하면서 발견했는데 아직 습관이 안되어서 몇개 빼먹었더라구요ㅎㅎ